spec.bs

```diff
@@ -177,9 +182,13 @@ dictionary GenerateBidInterestGroup {
 dictionary AuctionAdInterestGroup : GenerateBidInterestGroup {
   double priority = 0.0;
   record<DOMString, double> prioritySignalsOverrides;
+  DOMString? additionalBidKey;
 };
 </xmp>
 
+Note: An {{AuctionAdInterestGroup}} cannot include both {{GenerateBidInterestGroup/ads}} and

```

Comment:
We're adding one additional restriction to make negative IGs non-updatable: Could you please change this to something like, "An {{AuctionAdInterestGroup}} that includes {{AuctionAdInterestGroup/additionalBidKey}} cannot include either of {{GenerateBidInterestGroup/ads}} or {{GenerateBidInterestGroup/updateUrl}}"?